website/app.js

```diff
@@ -1,18 +1,44 @@
-const benchmarkNames = ["hello", "relative_import"];
-
-(async () => {
-  const data = await (await fetch("./data.json")).json();
+export async function getJson(path) {
+  return (await fetch(path)).json();
+}
```

Comment:
This constitutes the first multi-platform deno script, since it now runs in deno and the browser : )